rules/integrations/cloud_defend/privilege_escalation_debugfs_launched_inside_a_privileged_container.toml

```diff
@@ -0,0 +1,61 @@
+[metadata]
+creation_date = "2023/10/26"
+integration = ["cloud_defend"]
+maturity = "production"
+min_stack_comments = "New Integration: Cloud Defend"
+min_stack_version = "8.8.0"
+updated_date = "2023/12/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the use of the built-in Linux DebugFS utility from inside a privileged container. DebugFS is a special
+file system debugging utility which supports reading and writing directly from a hard drive device. When launched inside
+a privileged container, a container deployed with all the capabilities of the host machine, an attacker can access
+sensitive host level files which could be used for further privilege escalation and container escapes to the host
+machine.
+"""
+from = "now-6m"
+index = ["logs-cloud_defend*"]
+interval = "5m"
+language = "eql"
+license = "Elastic License v2"
+name = "File System Debugger Launched Inside a Privileged Container"
+references = [
+    "https://cyberark.wistia.com/medias/ygbzkzx93q?wvideo=ygbzkzx93q",
+    "https://book.hacktricks.xyz/linux-hardening/privilege-escalation/docker-security/docker-breakout-privilege-escalation#privileged",
+]
+risk_score = 73
```

Comment:
Might be good to start with a medium risk score, and move up in promotion when we are sure that there are limited FPs!